drizzlepac/haputils/hapcut_utils.py

```diff
@@ -437,50 +464,133 @@ def make_the_cut(input_files, sky_coord, cutout_size, single_outfile=True, outpu
-                          "d" + ra_frac + "-dec" + ns + str(dec_whole) + "d" + \
-                          dec_frac + ".fits"
 
-        cutout_path = os.path.join(output_dir, output_filename)
+#def mvm_combine(cutout_files, img_combiner=None, output_dir=".", log_level=logutil.logging.INFO):
+def mvm_combine(cutout_files, output_dir=".", log_level=logutil.logging.INFO):
+    """
+    This function combines multiple MVM skycell cutout images from the same detector/filter combination
+    to create a single view of the requested data.  All of the functions in this module are designed to
+    work in conjunction with one another, so the cutout images should be on the user's local disk.  This
+    task is a high-level wrapper for the `˜astrocut.cutout_processing.combine` functionality.
 
-        log.info("Cutout FITS filename: {}".format(cutout_path))
-    
-        # Get the primary header from the first input file, so we can retain
-        # a lot of information from at least the first observation for the PHDU
-        out_HDUList[0][0].header = fits.getheader(input_files[0])
+    Specifically, this routine will combine filter-level cutouts from multiple skycells, all sharing 
+    the same detector and filter.  This routine will also combine exposure-level cutouts from
+    multiple skycells, all sharing the same detector, filter, and ipppssoo.  Images which do not
+    share a detector and filter with any other image will be ignored. Individual exposures from
+    a single skycell will also be ignored.
+
+    Parameters
+    ----------
+    cutout_files : list
+        List of fits image cutout filenames where the cutouts are presumed to have been created
+        with `drizzlepac.haputils.hapcut_utils.make_the_cut`.
 
-        # Update PHDU FILENAME keyword with the new filename
-        out_HDUList[0][0].header["FILENAME"] = output_filename
+    # img_combiner : func 
+    #     The function to be used to combine the images
 
-        # Populate a PHDU HISTORY keyword with the names of all the input filenames
-        out_HDUList[0][0].header["HISTORY"] = input_files
+    output_dir : str
+        Default value '.'. The directory to save the cutout file(s) to.
 
-        out_HDUList[0].writeto(cutout_path, overwrite=True)
-        filename_list.append(output_filename)
+    log_level : int, optional
+        The desired level of verbosity in the log statements displayed on the screen and written to the
+        .log file. Default value is 20, or 'info'.
 
-    return filename_list
+    """
 
+    img_combiner = None
 
-def mvm_combine(cutout_files, img_combiner=None, output_dir=".", log_level=logutil.logging.INFO):
+    # set logging level to user-specified level
+    log.setLevel(log_level)
+
+    # Make sure the cutout_files are really a list of MULTIPLE filenames
+    if type(cutout_files) == str or type(cutout_files) == list and len(cutout_files) < 2:
```

Comment:
The next iteration of this function could support a filename as input, where the file provides the list of exposures to be combined.

Reply:
OK - will consider this.  This function was meant to accept as input the output from the make_the_cut routine which is a list of filenames.